README.md

```diff
@@ -90,6 +90,10 @@ Update to reflect compliance data released in December 2023.
 
 Update to reflect compliance data released in December 2024.
 
+### 4.0.0 (Dec 15, 2025)
```

Comment:
looks like there is drift in release # and the version number on your data? not a big deal but maybe there is a sane way to sync the two?